docs/basics/basic_benchmark.md

```diff
@@ -0,0 +1,139 @@
+# Basic Benchmark Structure
+
+
+Every benchmark starts with these global configuration.
+
+```yaml
+name: BENCHMARK_NAME
+http:
+  host: URL
+  sharedConnections: CONNECTIONS
+phases: ...
+```
+
+* URL is the HTTP protocol and hostname of the Application Under Test (e.g. http://myapp.com)
+* Shared connections are physical HTTP (1.x or2) connection.
```

Comment:
typo: s/or2/or 2/ However mentioning 1.x vs. 2 is confusing when you don't tell how to set one or another (which is fine at this point).

You had more info here, didn't you? I am missing at least saying '.. is the total **number** of HTTP connections`.

Reply:
I had more, but it was talking about memory pool, I took it out (original : Shared connections are allocated at the start time by keeping a pool of memory and initialized for the Virtual Users (VUs). When the VU finishes it execution, this memory is reset and reused for another VU.)